pkg/config/datafileprojectconfig/mappers/condition_trees_test.go

```diff
@@ -159,3 +159,28 @@ func TestBuildConditionTreeSimpleAudienceCondition(t *testing.T) {
 	}
 	assert.Equal(t, expectedConditionTree, conditionTree)
 }
+
+func TestBuildConditionTreeWithLeafNode(t *testing.T) {
+	conditionString := "{ \"type\": \"custom_attribute\", \"name\": \"s_foo\", \"match\": \"exact\", \"value\": \"foo\" }"
```

Comment:
@mikeng13 , I thought this case cannot be created by default, our app does not allow for that, is this the case ? 
if so then we should not support it.

Reply:
Yeah looks like this case is supported. I wasn’t aware of it before but we have an integration test around it which all other SDKs support

Reply:
Yeah looks like this case is supported. I wasn’t aware of it before but we have an integration test around it which all other SDKs support